install/ui/util/make-jquery.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -eu
+
+if [ $# -ne 1 ]; then
+    echo 'The script requires exactly one argument (a jQuery version):'
+    echo
+    echo '  $ ./make-jquery.sh 3.4.1'
+    echo
+    exit 1
+fi
+
+WD=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+JQUERY_VERSION=$1
+
+# Apply patches
+cd /tmp
+git clone -b ${JQUERY_VERSION} --depth 1 https://github.com/jquery/jquery.git
+cd jquery
+git am ${WD}/jquery-patches/${JQUERY_VERSION}/*
+
+# Build jQuery
+npm install
+npm run-script build
+
+# Replace the project version of jQuery with the built one
+cp -fv dist/jquery.min.js ${WD}/../src/libs/jquery.js
```

Comment:
You also need to clean up the resulted build tree and sources.

Reply:
Fixed

Reply:
Please change the name for `$TMPDIR` variable. `$TMPDIR` is actually a reserved name used in the system environment by many utilities to point to the current temporary directory. For example, `mktemp` uses it already.

Reply:
Changed